integrationtests/testdata/multiTransferWithScCallAndErrorSignaledBySC/transaction-after-execution-of-scr-dst-shard.json

```diff
@@ -6,10 +6,15 @@
   "data": "TXVsdGlFU0RUTkZUVHJhbnNmZXJAMDAwMDAwMDAwMDAwMDAwMDA1MDA1ZWJlYjM1MTVjYjQyMDU2YTgxZDQyYWRhZjc1NmEzZjYzYTM2MGJmYjA1NUAwMkA1NzQ1NDc0YzQ0MmQ2MjY0MzQ2NDM3MzlAQDM4ZTYyMDQ2ZmIxYTAwMDBANTg0ZDQ1NTgyZDY2NjQ2MTMzMzUzNUAwN0AwNDg5MDdlNTgyODRjMjhlODk4ZTI5QDYxNjQ2NDRjNjk3MTc1Njk2NDY5NzQ3OTUwNzI2Zjc4NzlAMDAwMDAwMDAwMDAwMDAwMDA1MDBlYmQzMDRjMmYzNGE2YjNmNmE1N2MxMzNhYjdiOGM2ZjgxZGM0MDE1NTQ4M0AzOGQ3OGY1OTU3ODVjMDAwQDA0ODdkZWFjMzEzYzZmNmIxMTE5MDY=",
   "signature": "",
   "fee": "1904415000000000",
+  "feeNum": 0.001904415,
   "esdtValues": [
     "4100000000000000000",
     "5482813151746926867746345"
   ],
+  "esdtValuesNum": [
+    4.1,
+    5.482813151746927e+06
```

Comment:
This is what I've talked about: discrepancy between `4.1` and `5.48 * 10^-6` 
This is just the way floats work, nothing to do about